typed-racket-lib/typed-racket/types/pairwise-intersect.rkt

```diff
@@ -0,0 +1,121 @@
+#lang racket
+
+;; This module provides a function pairwise-intersect for calculating a lower
```

Comment:
This file seems to raise a lot of errors instead of returning default results. Can you help me understand why?

Reply:
There's no good reason why it explicitly raises errors, so I can change it to either return default results or I can drop the catch-all cases.

Repeating what I put in one of the other comments: I initially implemented `pairwise-intersect` just to get a few examples working, and I thought raising an error might help me debug the cases I missed as I added more examples.